src/Apps/Artist/ArtistApp.tsx

```diff
@@ -49,33 +45,17 @@ export class ArtistApp extends React.Component<ArtistAppProps> {
           <Row>
             <Col>
               <NavigationTabs artist={artist} />
-
               <Spacer mb={3} />
 
-              {/*
-              When clicking nav links, wait for fetch to complete before
-              transitioning to new route
-            */}
-
-              <Subscribe to={[PreloadLinkState]}>
-                {({ state: { isLoading } }: PreloadLinkState) => {
-                  return (
-                    <LoadingArea isLoading={isLoading}>{children}</LoadingArea>
-                  )
-                }}
-              </Subscribe>
+              <Box minHeight="30vh">{children}</Box>
```

Comment:
This `30vh` is arbitrary, just enough to prevent the layout from shifting too much between tab transitions.